climada/hazard/tc_tracks.py

```diff
@@ -3094,3 +3094,196 @@ def _zlib_from_dataarray(data_var: xr.DataArray) -> bool:
     if np.issubdtype(data_var.dtype, float) or np.issubdtype(data_var.dtype, int):
         return True
     return False
+
+
+def compute_track_density(
```

Comment:
Why is this not a class method since the first argument seems to be the object itself?